bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/ZZjj_llvvjj_ewk_aQGC/ZZjj_llvvjj_ewk_aQGC_run_card.dat

```diff
@@ -0,0 +1,271 @@
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+  0    =  nhel           ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+  10.0    =  ptj        ! minimum pt for the jets 
+  10.0    =  ptb        ! minimum pt for the b 
+  0.0    =  pta        ! minimum pt for the photons 
+  0.0    =  ptl        ! minimum pt for the charged leptons 
+  0.0    =  misset     ! minimum missing Et (sum of neutrino's momenta)
+  0.0    =  ptheavy    ! minimum pt for one heavy final state
+  1.0   = ptonium     ! minimum pt for the quarkonium states
+  -1.0    =  ptjmax     ! maximum pt for the jets
+  -1.0    =  ptbmax     ! maximum pt for the b
+  -1.0    =  ptamax     ! maximum pt for the photons
+  -1.0    =  ptlmax     ! maximum pt for the charged leptons
+  -1.0    =  missetmax  ! maximum missing Et (sum of neutrino's momenta)
+#*********************************************************************
+# Minimum and maximum E's (in the center of mass frame)              *
+#*********************************************************************
+  0.0    =  ej      ! minimum E for the jets 
+  0.0    =  eb      ! minimum E for the b 
+  0.0    =  ea      ! minimum E for the photons 
+  0.0    =  el      ! minimum E for the charged leptons 
+  -1.0    =  ejmax  ! maximum E for the jets
+  -1.0    =  ebmax  ! maximum E for the b
+  -1.0    =  eamax  ! maximum E for the photons
+  -1.0    =  elmax  ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  6.5    =  etaj     ! max rap for the jets 
+  6.5    =  etab     ! max rap for the b
+  2.5    =  etaa     ! max rap for the photons 
+  -1.0    =  etal     ! max rap for the charged leptons 
+  0.6   = etaonium  ! max rap for the quarkonium states
+  0.0    =  etajmin  ! min rap for the jets
+  0.0    =  etabmin  ! min rap for the b
+  0.0    =  etaamin  ! min rap for the photons
+  0.0    =  etalmin  ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+  0.4    =  drjj     ! min distance between jets 
+  0.4    =  drbb     ! min distance between b's 
+  0.4    =  drll     ! min distance between leptons 
```

Comment:
Are you aware that your leptons from Z boson would have this cut applied. I would suggest to remove this as it seems a bit tight

Reply:
Actually every dR cut removal would be my suggestion

Reply:
removed all dR cuts